GOVERNANCE.md

```diff
@@ -1,114 +1,160 @@
+* Fairness: All stakeholders have the opportunity to provide feedback and submit contributions, which will be considered on their merits and not from source.
+* Community over Product or Company: Sustaining and growing our community takes priority over shipping code or sponsors' organizational goals.  Each
+  contributor participates in the project as an individual.
+* Prefer Stable Evolution: A willingness to constantly improve and reevaluate old decisions will keep the project and community relevant and growing.
+  However, this constant growth cannot be done at the expense of stability of the project. New features should be introduced liberally, but must be configured and opted-into until there is proven stability.
+* Inclusivity: We innovate through different perspectives and skill sets, which can only be accomplished in a welcoming and respectful environment.
+* Participation: Responsibilities within the project are earned through
+  participation, and there is a clear path up the contributor ladder into leadership positions.
+ 
+## Contribution Tiers
 
-* Anyone can and is encouraged to be a community member! The CRI-O community eagerly accept PRs, issues, new users and questions.
+CRI-O has four tiers of code contributors: community members, reviewers, approvers and emeritus approvers.
+* Anyone can (and is encouraged to) be a community member, whose responsibilities are up to the contributor (submitting PRs, reviewing PRs).
+* Reviewers are a set of people responsible for reviewing PRs and triaging issues. In most cases, they're also responsible for fixing issues in the code base.
+* Approvers are the set of people that have final say over the direction of the project. For any PR to merge, an approver must approve it.
+* Emeritus Approvers can be considered "retired" approver--while they don't have approval rights, they are listed in the OWNERS file to indicate their expertise in the area.
+The current list of reviewers and approvers can be found in the [CRI-O OWNERS file](https://github.com/cri-o/cri-o/blob/main/OWNERS).
+Further, some more detailed information can be found in the [CRI-O Maintainers file](https://github.com/cri-o/cri-o/blob/main/MAINTAINERS.md).
 
-## Reviewer
+### Community Member
 
-### How to become a reviewer
+* Anyone can and is encouraged to be a community member! The CRI-O community eagerly accepts PRs, issues, new users and questions.
+
+### Reviewer
+
+#### How to become a reviewer
 
 * If a community member would like to take on a more active role in the community, and one day become an approver, becoming a reviewer is a good first step.
 * Becoming a reviewer allows someone to build trust with the community and deepen their knowledge of the code base.
 * To become a reviewer, one should file an issue with [this template](https://github.com/cri-o/cri-o/issues/new?assignees=&labels=area%2Fgithub-membership&template=membership.yml&title=REQUEST%3A+New+membership+for+%3Cyour-GH-handle%3E).
```

Comment:
Looks like the `membership.yml` template is no longer there, use the following URL to leverage the current template.

```
https://github.com/cri-o/cri-o/issues/new?template=reviewer.yml&title=REQUEST%3A+New+reviewer+status+for+%3Cyour-GH-handle%3E
```